### ISSUE TITLE
asyncio warning on startup / spam on shutdown.

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-sdk#2265]
startup: 
```
2019-06-25 09:54:38,096 INFO     lbry.extras.daemon.Components:258: Starting torba wallet
2019-06-25 09:54:38,151 INFO     lbry.extras.daemon.Components:336: start the dht
2019-06-25 09:54:38,152 WARNING  lbry.extras.daemon.Components:342: UPnP component failed to get external ip
2019-06-25 09:54:38,163 INFO     torba.client.basedatabase:208: connecting to database: c:\testing\lbrytv\wallet\lbc_mainnet\blockchain.db
2019-06-25 09:54:38,263 INFO     lbry.extras.daemon.Components:361: Started the dht
2019-06-25 09:54:38,268 INFO     lbry.dht.node:118: DHT node listening on UDP 0.0.0.0:4444
2019-06-25 09:54:38,284 DEBUG    lbry.dht.protocol.iterative_find:267: probing 18.213.182.111:4444
2019-06-25 09:54:38,286 DEBUG    lbry.dht.protocol.iterative_find:267: probing 35.161.227.101:4444
2019-06-25 09:54:38,289 DEBUG    lbry.dht.protocol.iterative_find:267: probing 3.122.90.177:4444
2019-06-25 09:54:38,291 DEBUG    lbry.dht.protocol.iterative_find:267: probing 52.78.118.133:4444
Task was destroyed but it is pending!
source_traceback: Object created at (most recent call last):
  File "C:\Users\thoma\Documents\lbry\lbry-venv\Scripts\lbrynet-script.py", line 11, in <module>
    load_entry_point('lbry', 'console_scripts', 'lbrynet')()
  File "c:\users\thoma\documents\lbry\lbry\lbry\extras\cli.py", line 287, in main
    loop.run_until_complete(daemon.start())
  File "C:\Users\thoma\AppData\Local\Programs\Python\Python37-32\lib\asyncio\base_events.py", line 571, in run_until_complete
    self.run_forever()
  File "C:\Users\thoma\AppData\Local\Programs\Python\Python37-32\lib\asyncio\base_events.py", line 539, in run_forever
    self._run_once()
  File "C:\Users\thoma\AppData\Local\Programs\Python\Python37-32\lib\asyncio\base_events.py", line 1767, in _run_once
    handle._run()
  File "C:\Users\thoma\AppData\Local\Programs\Python\Python37-32\lib\asyncio\events.py", line 88, in _run
    self._context.run(self._callback, *self._args)
  File "c:\users\thoma\documents\lbry\torba\torba\client\basenetwork.py", line 205, in ensure_connections
    await asyncio.wait([asyncio.sleep(3), self._lost_master.wait()], return_when='FIRST_COMPLETED')
  File "C:\Users\thoma\AppData\Local\Programs\Python\Python37-32\lib\asyncio\tasks.py", line 361, in wait
    fs = {ensure_future(f, loop=loop) for f in set(fs)}
  File "C:\Users\thoma\AppData\Local\Programs\Python\Python37-32\lib\asyncio\tasks.py", line 361, in <setcomp>
    fs = {ensure_future(f, loop=loop) for f in set(fs)}
task: <Task pending coro=<Event.wait() running at C:\Users\thoma\AppData\Local\Programs\Python\Python37-32\lib\asyncio\locks.py:293> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x05D9FCB0>()] created at C:\Users\thoma\AppData\Local\Programs\Python\Python37-32\lib\asyncio\base_events.py:396> cb=[_wait.<locals>._on_completion() at C:\Users\thoma\AppData\Local\Programs\Python\Python37-32\lib\asyncio\tasks.py:440] created at C:\Users\thoma\AppData\Local\Programs\Python\Python37-32\lib\asyncio\tasks.py:361>
2019-06-25 09:54:38,292 DEBUG    lbry.dht.protocol.iterative_find:192: running 4 probes
2019-06-25 09:54:38,318 INFO     torba.client.basenetwork:89: Successfully connected to SP
```
Shutdown: ```019-06-25 01:57:48,687 INFO     lbry.extras.daemon.Daemon:471: finished shutting down
Task was destroyed but it is pending!
source_traceback: Object created at (most recent call last):
  File "lbry\extras\cli.py", line 324, in <module>
  File "lbry\extras\cli.py", line 287, in main
  File "asyncio\base_events.py", line 571, in run_until_complete
  File "asyncio\base_events.py", line 539, in run_forever
  File "asyncio\base_events.py", line 1767, in _run_once
  File "asyncio\events.py", line 88, in _run
  File "torba\client\basenetwork.py", line 205, in ensure_connections
  File "asyncio\tasks.py", line 361, in wait
  File "asyncio\tasks.py", line 361, in <setcomp>
task: <Task pending coro=<Event.wait() running at asyncio\locks.py:293> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x08159DB0>()] created at asyncio\base_events.py:396> cb=[_wait.<locals>._on_completion() at asyncio\tasks.py:440] created at asyncio\tasks.py:361>
Task was destroyed but it is pending!
source_traceback: Object created at (most recent call last):
  File "lbry\extras\cli.py", line 324, in <module>
  File "lbry\extras\cli.py", line 289, in main
  File "asyncio\base_events.py", line 571, in run_until_complete
  File "asyncio\base_events.py", line 539, in run_forever
  File "asyncio\base_events.py", line 1767, in _run_once
  File "asyncio\events.py", line 88, in _run
  File "torba\client\basenetwork.py", line 205, in ensure_connections
  File "asyncio\tasks.py", line 361, in wait
  File "asyncio\tasks.py", line 361, in <setcomp>
task: <Task pending coro=<Event.wait() done, defined at asyncio\locks.py:280> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x08186DF0>()] created at asyncio\base_events.py:396> cb=[_wait.<locals>._on_completion() at asyncio\tasks.py:440] created at asyncio\tasks.py:361>
Task was destroyed but it is pending!
source_traceback: Object created at (most recent call last):
  File "lbry\extras\cli.py", line 324, in <module>
  File "lbry\extras\cli.py", line 287, in main
  File "asyncio\base_events.py", line 571, in run_until_complete
  File "asyncio\base_events.py", line 539, in run_forever
  File "asyncio\base_events.py", line 1767, in _run_once
  File "asyncio\events.py", line 88, in _run
  File "torba\client\basenetwork.py", line 205, in ensure_connections
  File "asyncio\tasks.py", line 361, in wait
  File "asyncio\tasks.py", line 361, in <setcomp>
task: <Task pending coro=<Event.wait() running at asyncio\locks.py:293> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x08166150>()] created at asyncio\base_events.py:396> created at asyncio\tasks.py:361>
 ```

**Comment by tzarebczan** [comment]
This is still happening, if it's an easy one, let's please fix (@shyba )

**Comment by lyoshenka** [comment]
i think i fixed this in #2297 too (h/t @jackrobison and @shyba for the help)
